Stable/v1.9.x

bermudezjd opened the pull request:
Run Django 1.9 from master.  Adds `init_postgres_db` management command and cleans up a few other settings.

@cmurtaugh 
@rascalking 
@elliottyates 
@eparker71 
@sapnamysore

bermudezjd committed:
Declare SECRET_KEY in both local and aws settings file so that we can trigger an error during deployment if it's not set there.  Similarly, set DEBUG explicitly to True in local settings and pull from environment vars in aws settings.  No need to set the debug key in TEMPLATES config since it defaults to the value of DEBUG.

bermudezjd committed:
Use Django 1.8.6 instead of 1.8.4

bermudezjd committed:
Make `enable_debug` flag a required environment setting when deploying to AWS.

cmurtaugh committed:
Update Django version to 1.9.1

cmurtaugh committed:
Update debug-toolbar to 1.4 (compatible w/Django 1.9+)

cmurtaugh committed:
Properly add debug_toolbar to installed apps/middleware

cmurtaugh committed:
Update the readme to be version specific

Since this will live in its own stable branch we can hardcode the
version number

cmurtaugh committed:
Fixed permissions on env_bootstrap.sh

cmurtaugh committed:
Updated redis version

bermudezjd committed:
Update redis and Django versions.  Provide a valid secret_key when running locally.

bermudezjd committed:
Merge branch 'stable/v1.8.x' into stable/v1.9.x

Conflicts:
	project_name/requirements/base.txt

Updated 1.9.x to include latest from 1.8.x minus Django 1.8.8 :)

bermudezjd committed:
Add project-level management command that initializes the default postgres db for a project.  Include project as an installed app to allowed this command to be called (with local settings).  Remove ALLOWED_HOSTS declaration from local settings, since docs say that host checking is disabled when DEBUG is set to True.

bermudezjd committed:
Update Django version.  Replace 1.8 refs with 1.9.

cmurtaugh commented:
Looks good -- might want to document the reasoning behind adding the project_name to the installed apps list. :+1:

bermudezjd committed:
Removing init_postgres_db management command in favor of matching shell script in tlt-vagrant

Removes init_postgres_db management command as well as the inclusion of
the project directory as an installed app.

bermudezjd commented:
I ended up removing the `init_postgres_db` command from the template (in favor of a shell script elsewhere), so adding the project folder as an installed app is no longer necessary.
